src/display/annotation_layer.js

```diff
@@ -1901,7 +1902,7 @@ class PopupElement {
     }
     if (this.hideElement.hidden) {
       this.hideElement.hidden = false;
-      this.container.style.zIndex += 1;
+      this.container.style.zIndex += 10000;
```

Comment:
Couldn't such a large value cause a Popup to "interfere" with UI elements such as the findbar and secondaryToolbar (given their z-index definitions)?
You might want to limit this value, and the one below, to only `1000` instead since the likelihood of any one page having (even close to) that many annotation should be extremely low.